internal/pkg/manifest/svc.go

```diff
@@ -26,6 +27,50 @@ var ServiceTypes = []string{
 	BackendServiceType,
 }
 
+// RangeOpts contains either a Range or a range configuration for Autoscaling ranges
+type RangeOpts struct {
+	Range       *Range // Mutually exclusive with RangeConfig
+	RangeConfig RangeConfig
+}
```

Comment:
What do you think of this to follow after what we did with the `Count` type?
```suggestion
// RangeOpts contains either a Range or a range configuration for Autoscaling ranges
type Range struct {
	Value       *IntRangeBand
	Advanced    AdvancedRange
}
```

Reply:
https://github.com/aws/copilot-cli/pull/2188/commits/a4d886e3eb1a357a6668e1a3b1994bb991021b6a